nexuse2e-webapp/src/main/frontend/src/app/messages-failed/messages-failed.component.ts

```diff
@@ -12,12 +17,30 @@ export class MessagesFailedComponent implements OnInit {
   @Input() dashboardTimeFrameInDays = 0;
   failedMessages: Message[] | undefined;
   failedMessagesConfig = FAILED_MESSAGES_CONFIG;
+  mobileCardLinkUrl = CARD_LINK_CONFIG.linkUrl;
+  mobileCardLinkQueryParams = CARD_LINK_CONFIG.linkParamsRecipe;
+  failedMessagesCount = 0;
   loaded = false;
 
-  constructor(private dataService: DataService) {}
+  constructor(
+    private dataService: DataService,
+    public screenSizeService: ScreensizeService,
+    public requestHelper: RequestHelper
```

Comment:
Can this Service be private?
```suggestion
    private requestHelper: RequestHelper
```

Reply:
No, because it is used in the html file.